### DISCUSSION TITLE
Whether the channel generation result of Sionna is compared with the simulation result of the proposals submitted by each company to 3GPP?

### DISCUSSION THREAD

**Discussion by majestichou** [NVlabs/sionna#998] [General]
Whether the channel generation result of Sionna is compared with the simulation result of the proposals submitted by each company to 3GPP?

**Comment by majestichou** [comment]
Such as CDL Channel results?

**Comment by jhoydis** [comment]
Hi @majestichou,

We have not done such a comparison, yet. Is there a precise document that you are referring to?

**Comment by majestichou** [comment]
@jhoydis 
The link to the channel calibration reference document for system simulation is [channel calibration reference file](https://www.etsi.org/deliver/etsi_tr/138900_138999/138901/16.01.00_60/tr_138901v160100p.pdf).  The configuration and results of channel calibration of each company are provided in the document. You can pay attention to a passage in the file.

```
The calibration parameters for full calibration including the fast fading modelling can be found in Table 7.8-2.
Unspecified parameters in Table 7.8-2 are the same as those in Table 7.8-1. When P=2, X-pol (+/-45 degree) is used for
BS antenna configuration 1 and X-pol (0/+90 degree) is used for UT antenna configuration. The calibration results
based on TR 38.900 V14.0.0 can be found in R1-165975
```

**Comment by budavarapu** [comment]
We can check Quadriga calibration results and use for comparison
